components/os/unix.go

```diff
@@ -45,6 +53,14 @@ func getDefaultInstanceType(env config.Environment, arch Architecture) string {
 }
 
 func getUnixInstallFormatString(scriptName string, version AgentVersion) string {
+	if version.IsCustomImage {
+		commandLine := fmt.Sprintf(`TESTING_APT_URL=apttesting.datad0g.com TESTING_APT_REPO_VERSION="%v %v" `, version.RepoBranch, getAgentMajorVersion(version))
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Following @bkabrda hint, this suggestion would make the script work with all linux versions 

```suggestion
		testEnvVars := []string{}
		testEnvVars = append(testEnvVars, "TESTING_APT_URL=apttesting.datad0g.com")
		testEnvVars = append(testEnvVars, fmt.Sprintf(`TESTING_APT_REPO_VERSION="%v %v"`, version.RepoBranch, getAgentMajorVersion(version)))
		testEnvVars = append(testEnvVars, "TESTING_YUM_URL=yumtesting.datad0g.com")
		testEnvVars = append(testEnvVars, fmt.Sprintf("TESTING_YUM_VERSION_PATH=testing/%v/%v", version.RepoBranch, getAgentMajorVersion(version)))
		commandLine := strings.Join(testEnvVars, " ")
```

Reply:
⚠️ my suggestion above would break the code, you'll need to add import of `strings`, sorry for this

Reply:
changed and added the needed import, thanks :)